<ui version="4.0" >
 <class>StatusLayout</class>
 <widget class="QWidget" name="StatusLayout" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLineEdit" name="filterEdit" />
   </item>
   <item>
    <widget class="QWidget" native="1" name="widget" >
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="modifiedCheckBox" >
        <property name="toolTip" >
         <string>Show or Hide Modified Files</string>
        </property>
        <property name="text" >
         <string>M</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="addedCheckBox" >
        <property name="toolTip" >
         <string>Show or Hide Added Files</string>
        </property>
        <property name="text" >
         <string>A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="removedCheckBox" >
        <property name="toolTip" >
         <string>Show or Hide Removed Files</string>
        </property>
        <property name="text" >
         <string>R</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="deletedCheckBox" >
        <property name="toolTip" >
         <string>Show or Hide Deleted Files</string>
        </property>
        <property name="text" >
         <string>!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="unknownCheckBox" >
        <property name="toolTip" >
         <string>Show or Hide Files not tracked by the repository</string>
        </property>
        <property name="text" >
         <string>?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ignoredCheckBox" >
        <property name="toolTip" >
         <string>Show or Hide Ignored Files</string>
        </property>
        <property name="text" >
         <string>I</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cleanCheckBox" >
        <property name="toolTip" >
         <string>Show or Hide Files with no changes.</string>
        </property>
        <property name="text" >
         <string>C</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>filterEdit</zorder>
     <zorder>comboBox</zorder>
     <zorder>modifiedCheckBox</zorder>
     <zorder>ignoredCheckBox</zorder>
     <zorder>unknownCheckBox</zorder>
     <zorder>addedCheckBox</zorder>
     <zorder>deletedCheckBox</zorder>
     <zorder>removedCheckBox</zorder>
     <zorder>cleanCheckBox</zorder>
     <zorder>scrollArea</zorder>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea" >
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable" >
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>-126</y>
        <width>420</width>
        <height>461</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QWidget" native="1" name="parent1Container" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" native="1" name="widget_5" >
            <layout class="QHBoxLayout" name="horizontalLayout_3" >
             <item>
              <widget class="QLabel" name="label_2" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text" >
                <string>Changes From Revision:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="parent1Label" >
               <property name="text" >
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="parent1TableView" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="MinimumExpanding" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="verticalScrollBarPolicy" >
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="alternatingRowColors" >
             <bool>false</bool>
            </property>
            <property name="selectionMode" >
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior" >
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="showGrid" >
             <bool>false</bool>
            </property>
            <property name="sortingEnabled" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" native="1" name="parent2Container" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="sizePolicy" >
          <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="Line" name="line" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" native="1" name="widget_2" >
            <layout class="QHBoxLayout" name="horizontalLayout_2" >
             <item>
              <widget class="QLabel" name="label" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text" >
                <string>Changes From Revision:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="parent2Label" >
               <property name="text" >
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="parent2TableView" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="MinimumExpanding" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="verticalScrollBarPolicy" >
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="alternatingRowColors" >
             <bool>false</bool>
            </property>
            <property name="selectionMode" >
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior" >
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="showGrid" >
             <bool>false</bool>
            </property>
            <property name="sortingEnabled" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
